CHANGELOG.md

```diff
@@ -56,12 +65,6 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Save license when a video is created
 - Show an error when a resource has been deleted
 - Allow depositedfiles to have unicode characters in their filename
-- Correctly redirect a user when a resource is deleted (#2419)
-- Checkbox to delete resource remains displayed on others pages (#2416)
-- Remove a non-existent attribute which breaks classroom document admin view
-- Change color of a disabled button in the deposit file student dashboard
-- Remove multiple primary buttons in the deposit file dashboard
-- Remove multiple calls of playlist is-claimed endpoint
```

Comment:
Those lines should be kept here.

Reply:
Ah yes you'r right, I thought the [release](https://github.com/openfun/marsha/releases) one was the good one.